chain/validation/lotus_test.go

```diff
@@ -51,3 +57,84 @@ func TestLotusExample(t *testing.T) {
 	drv.AssertBalance(miner, uint64(expectedGasUsed))
 
 }
+func TestLotusCreateStorageMiner(t *testing.T) {
+	factory := NewFactories()
+	drv := suites.NewStateDriver(t, factory.NewState())
+	gasPrice := big.NewInt(1)
+	gasLimit := state.GasUnit(2000) // needs to be over ~1100 for lotus operations
```

Comment:
I'd set this to something really high for now (like 1000000). Gas is going to see some signigicant changes soon-ish